src/sql/workbench/contrib/objectExplorer/test/browser/utils.test.ts

```diff
@@ -0,0 +1,61 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as assert from 'assert';
+import { ConnectionProfileGroup } from 'sql/platform/connection/common/connectionProfileGroup';
+import { ConnectionProfile } from 'sql/platform/connection/common/connectionProfile';
+import { TestCapabilitiesService } from 'sql/platform/capabilities/test/common/testCapabilitiesService';
+import { mssqlProviderName } from 'sql/platform/connection/common/constants';
+import * as ConnectionUtils from 'sql/platform/connection/common/utils';
+
+suite('Connection Utilities tests', () => {
+	let capabilitiesService = new TestCapabilitiesService();
+	let connection = new ConnectionProfile(capabilitiesService, {
+		connectionName: 'Test',
+		savePassword: false,
+		groupFullName: 'testGroup',
+		serverName: 'testServerName',
+		databaseName: 'master',
+		authenticationType: 'inetgrated',
+		password: 'test',
+		userName: 'testUsername',
+		groupId: undefined,
+		providerName: mssqlProviderName,
+		options: {},
+		saveProfile: true,
+		id: 'testID'
+	});
+
+	test('findProfileInGroup - test if utils can find profile in group', () => {
+		let conProfGroup = new ConnectionProfileGroup('testGroup', undefined, 'testGroup', undefined, undefined);
+		let conProfGroupChild = new ConnectionProfileGroup('testGroupChild', conProfGroup, 'testGroupChild', undefined, undefined);
+		conProfGroup.addGroups([conProfGroupChild]);
+		conProfGroupChild.addConnections([connection]);
+		assert.equal(connection, ConnectionUtils.findProfileInGroup(connection, [conProfGroup]));
+	});
+
+	test('getUriPrefix - test if getUriPrefix finds the correct prefix from fake uri name', () => {
+		let testUri = 'test://testpath';
```

Comment:
An interesting test to add here would be to test an invalid uri and make sure it handles that.